src/components/profile/profile-header.tsx

```diff
@@ -29,37 +29,61 @@ export const ProfileHeader: React.FC<ProfileHeaderProps> = ({
 
   return (
     <>
-      <div className='flex justify-center'>
-        <Avatar className='h-20 w-20'>
-          <AvatarImage
-            src={user.profileImage || '/avatar.svg'}
-            alt={user.accountId}
-          />
-          <AvatarFallback className='text-2xl font-semibold'>
-            {user.accountId.slice(0, 2).toUpperCase()}
-          </AvatarFallback>
-        </Avatar>
-      </div>
-      <div className='flex justify-center'>
-        {user.firstName && user.lastName && (
-          <h1 className='text-2xl font-semibold'>{`${user.firstName} ${user.lastName}`}</h1>
-        )}
-      </div>
-      <div className='flex justify-center'>
-        <div className='lg:w-5/12'>
-          <h1 className='text-center text-3xl font-semibold'>{user.bio}</h1>
+      <div className='flex flex-col items-center gap-2'>
+        {/* Avatar start */}
+        <div className='flex justify-center'>
+          <Avatar className='h-[100px] w-[100px]'>
+            <AvatarImage
+              src={user.profileImage || '/avatar.svg'}
+              alt={user.accountId}
+            />
+            <AvatarFallback className='text-2xl font-semibold'>
```

Comment:
```suggestion
<AvatarFallback>
    <User01 className='text-muted-foreground' />
</AvatarFallback>
```

Let's move to using something like this for the fallback, You may need to adjust the colors and size of the icon to get it to match design.